cmd/add.go

```diff
@@ -259,7 +259,8 @@ space usage checking and learning only occurs for jobs where cwd doesn't matter
 "queue" tells wr which queue a job should be submitted to, when using a job
 scheduler that has queues (eg. LSF). If queue is not specified, wr will use
 heuristics to pick the most appropriate queue based on the time, memory and cpu
-requirements of the job.
+requirements of the job. Can be specified as a comma separated list to allow the
```

Comment:
This explanation is not clear/accurate enough. It doesn't "allow" the use, it restricts the use. See my previous review comment.